proto/redpanda/api/dataplane/v1alpha1/topic.proto

```diff
@@ -68,30 +73,30 @@ message CreateTopicRequest {
     // NumPartitions must be -1.
     repeated ReplicaAssignment replica_assignments = 4;
```

Comment:
@weeco For this field is it correct to say "Manually specify broker assignments for partition replicas"?

Reply:
Yes, but possibly clarify that we are referring to broker ids there, so that it's clear what exactly has to be used for the assignments